sdk/src/transaction/sanitized.rs

```diff
@@ -199,13 +203,8 @@ impl SanitizedTransaction {
         &self,
         tx_account_lock_limit: usize,
     ) -> Result<TransactionAccountLocks> {
-        if self.message.has_duplicates() {
-            Err(TransactionError::AccountLoadedTwice)
-        } else if self.message.account_keys().len() > tx_account_lock_limit {
-            Err(TransactionError::TooManyAccountLocks)
-        } else {
-            Ok(self.get_account_locks_unchecked())
-        }
+        Self::validate_account_locks(self.message(), tx_account_lock_limit)?;
```

Comment:
any thoughts on removing this since we check this when we create the `SanitizedTransaction`?

Reply:
there are few other public functions can create `SanitizedTransaction`, maybe call `validate_account_locks()` everywhere?

Reply:
working on it